docs/contributing/adding-images-to-the-wesite.md

```diff
@@ -0,0 +1,32 @@
+# Adding images to the design system website
+When adding or amending guidance on the design system website and you want to add an image, it's important to be considerate of the impact images have on the website. They can have implications for web performance and accessibility if used improperly.
+
+Before reading the following, also consult our [image guidance](https://design-system.service.gov.uk/styles/images/) for details on when it's appropriate to use images and how best to use them.
+
+## Does your content really need images?
+Generally, we would avoid adding new images to our content.
+
+In patterns and components, we should be using code examples when we want to convey what our pattern or component will look like. Avoid using screenshots of patterns or components wherever possible.
+
+## Can your image be an SVG?
+SVG's are usually better for web performance as they scale with different screen sizes and load faster than most image formats of the same size. If your image is a "graphic" (in other words it isn't a photo or a screenshot) it should be converted to an SVG and added to the page directly.
```

Comment:
```suggestion
SVG files are usually better for web performance as they scale with different screen sizes and load faster than most image formats of the same size. If your image is a "graphic" (in other words it isn't a photo or a screenshot) you should convert it to an SVG and add it directly to the page.
```

Reply:
Romaric has completely re-written this line, with some edits from me. Sorry to waste your time!